### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Kalunge**
## Bookstore: styling
in this milestone I:
- added styling to core components

### CODE REVIEW

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @Kalunge  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below make all required changes.

- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).
- [ ] It will be nice to stick to the project design(especially the nav and display for book item)
Screenshot from 2021-09-30 11-28-59

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 1]
```diff
@@ -56,26 +65,243 @@ li {
...
 
+.add-book {
+  width: 40rem;
+  margin-left: 10%;
+  height: 2.813rem;
+  margin-top: 1.188rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
```
Comment (Emmyn5600) [inline src/index.css:77]: - [ ] Kindly add `cursor:pointer` to make a button to be pointer

File: src/index.css [hunk 2]
```diff
@@ -56,26 +65,243 @@ li {
...
+  font-family: "RobotoSlab", sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
```
Comment (Emmyn5600) [inline src/index.css:126]: - [ ]  Kindly add `cursor:pointer` to make a button to be pointer

File: src/index.css [hunk 3]
```diff
@@ -6,6 +12,8 @@
   border-radius: 4px;
   box-shadow: 0 0 60px 0 var(--black-5);
   background-color: #fafafa;
+  font-size: 20px;
+  text-decoration: none;
 }
```
Comment (Emmyn5600) [inline src/index.css:18]: - [ ] Kindly, stick to the design, as you can use in the original design we don't have `Bookstore-CMS` underlined and space with book menu

rr26

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Kalunge 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

     Changes required :recycle: 
    
    -  Please check the comment under the review too

    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.css (hunk 3 above)
Comment (Kalunge) [reply src/index.css:18]: Hello, Emmy please note that my name is not black-box. I have implemented the requested changes

**Review by EricMbouwe — APPROVED** [review]
Hi @Kalunge, 

# Status: Approved✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**